iaso/gpkg/import_gpkg.py

```diff
@@ -404,6 +405,9 @@ def import_gpkg_file2(
         ou.parent = parent_ou
         ou.source_ref = ou.source_ref.replace(OLD_INTERNAL_REF, NEW_INTERNAL_REF)
         ou.save()
+
+    recalculate_missing_paths(version, task)
```

Comment:
I'm quite surprised this is needed in all cases. This has been working for sure in the past. Did you understand how this bug appeared?